src/Polysemy.hs

```diff
@@ -14,6 +14,10 @@ module Polysemy
   , Embed (..)
   , embed
   , embedToFinal
+
+  -- * Type synonyms for user convenience
+  -- ** InterpreterFor
+  , InterpreterFor
```

Comment:
This is exported in the wrong place; in the middle of `Interoperating with other monads`, which will make the documentation look weird. Instead, place this after the export of `runFinal`.

Also, you don't need the `InterpreterFor` subheading.

Reply:
Okay Thanks! I wasn't sure where to export it. I will correct this!